client/package.json

```diff
@@ -45,6 +45,7 @@
     "load-grunt-config": "0.17.1",
     "load-grunt-tasks": "3.2.0",
     "lodash": "2.4.2",
+    "phantomjs": "^2.1.3",
```

Comment:
Without this dependency, the branch was not deployed in Bamboo. It kept showing me this error: "UNMET PEER DEPENDENCY phantomjs@>=1.9"